src/core/openthread.cpp

```diff
@@ -876,6 +878,32 @@ ThreadError otRemoveUnicastAddress(otInstance *, const otIp6Address *address)
     return sThreadNetif->RemoveExternalUnicastAddress(*static_cast<const Ip6::Address *>(address));
 }
 
+void otSlaacUpdate(otInstance *aInstance, otNetifAddress *aAddresses, uint32_t aNumAddresses,
+                   otSlaacIidCreate aIidCreate, void *aContext)
+{
+    Utils::Slaac::updateAddresses(aInstance, aAddresses, aNumAddresses, aIidCreate, aContext);
+}
+
+ThreadError otCreateRandomIid(otNetifAddress *aAddress, void *aContext)
+{
+    return Utils::Slaac::createRandomIid(aAddress, aContext);
+}
+
+ThreadError otCreateMacIid(otNetifAddress *aAddress, void *aContext)
+{
+    (void) aContext;
+
+    memcpy(&aAddress->mAddress.mFields.m8[8], sThreadNetif->GetMac().GetExtAddress(), 8);
```

Comment:
Because you are accessing the static `sThreadNetif` here, you need the `otInstance*` passed in as well.